pkg/monitoring/perfscale/vmi-phase-transitions.go

```diff
@@ -129,11 +131,15 @@ func newVMIPhaseTransitionTimeHistogramVec(informer cache.SharedIndexInformer) *
 }
 
 func updateVMIPhaseTransitionTimeFromCreationTimeHistogramVec(histogramVec *prometheus.HistogramVec, oldVMI *v1.VirtualMachineInstance, newVMI *v1.VirtualMachineInstance) {
+	if newVMI.IsMarkedForDeletion() {
+		return
+	}
+
```

Comment:
Pulled this out.  If you use the Failed transition with the creation seconds metric it can be used for the full e2e time or the amount of time the VM spent in running.